modules/axhal/src/platform/aarch64_qemu_virt/mp.rs

```diff
@@ -6,5 +6,10 @@ pub fn start_secondary_cpu(cpu_id: usize, stack_top: PhysAddr) {
         fn _start_secondary();
     }
     let entry = virt_to_phys(VirtAddr::from(_start_secondary as usize));
-    crate::platform::aarch64_common::psci::cpu_on(cpu_id, entry.as_usize(), stack_top.as_usize());
+    crate::platform::aarch64_common::psci::psci_cpu_on(
+        false,
```

Comment:
PSCI cpu on code for QEMU virt arm should be `0xc4000003` (64bit), the origin code has a bug.